pkg/position_store/cache.go

```diff
@@ -15,12 +14,14 @@ type PositionCacheInterface interface {
 	Start() error
 	Close()
 	Put(position Position) error
+
 	// Get will get a value from cache, if there is no value inside the cache
 	// it will try to get it from position repo
 	Get() (position Position, exist bool, err error)
-	// Get will get a raw value from cache, if there is no value inside the cache
+
+	// GetWithValueString will serialize the Value to ValueString, if there is no value inside the cache
 	// it won't try to get it from position repo
-	GetWithRawValue() (position *PositionWithValueString, exist bool, err error)
+	GetWithValueString() (position Position, exist bool, err error)
```

Comment:
why `GetWithValueString` won't query repo but `Get` will? The api should be consistent. The should both query repo, or not, or providing option, or providing independent existence test method.